src/collar/oc_bookmarks.lsl

```diff
@@ -108,10 +110,6 @@ PermsCheck() {
 
 UserCommand(integer iNum, string sStr, key kID) {
     // So commands can accept a value
-    if (iNum==CMD_GROUP) {  // Do not permit Group Access (Public Access)
```

Comment:
This change looks more substantive than just ensuring cross-grid compatibility.  If we want to change the group access rules I think that should be a separate ticket and pull request.

Reply:
Oh I see maybe you moved this lower in the function.  Can you comment on the motivation for that, and how equivalent the new code is?

Reply:
I didn't think I changed this function, maybe a throwback somewhere, this is where GIT is useful